mlir/include/mlir/Dialect/MIOpen/MIOpenOps.td

```diff
@@ -292,6 +347,9 @@ def MIOpen_ThreadwiseLoadOp:
     allowed:
     - Global (generic tensor) to register (naive tensor).
   }];
+  let assemblyFormat = [{
+    `(` operands `)` attr-dict `:` type($source) `,` type($sourceCoord) `->` type($result)
```

Comment:
it's nice to have the type of the result be printed in the assembly. thanks!

Reply:
No problem!